torch_em/transform/generic.py

```diff
@@ -129,11 +134,23 @@ def _get_preprocess_shape(self, oldh, oldw):
         newh = int(newh + 0.5)
         return (newh, neww)
 
+    def convert_transformed_inputs_to_original_shape(self, resized_inputs):
+        if not hasattr(self, "pre_pad_shape"):
+            raise AttributeError(
```

Comment:
`AttributeError` doesn't really make sense for this. I would rather throw a `RuntimeError`.